src/background/Wallet/Wallet.ts

```diff
@@ -799,6 +805,29 @@ export class Wallet {
     chainConfigStore.setDefiSdkClient(client);
   }
 
+  private updateLastAddress() {
+    const currentAddress = this.readCurrentAddress();
+    if (this.id && currentAddress) {
+      lastUsedAddress.setState({
+        address: currentAddress,
+        walletModelId: this.id,
```

Comment:
Is there a case when we request this address with another userId?
As I understand, it can happen after creating new user which should lead to clearing the whole data anyway.

Reply:
I'm just following the idea that current user id and wallet model id must match